packages/host/tests/integration/search-index-test.ts

```diff
@@ -2487,6 +2487,21 @@ posts/please-ignore-me.json
     }
   });
 
+  test('search index ignored .realm.json file', async function (assert) {
+    let realm = await TestRealm.create(
+      {
+        '.realm.json': `{ name: 'Example Workspace' }`,
+        'post.json': { data: { meta: { adoptsFrom: baseCardRef } } },
+      },
+      this.owner
+    );
+
+    await realm.ready;
+    let indexer = realm.searchIndex;
+    let card = await indexer.card(new URL(`${testRealmURL}post`));
+    assert.ok(card, 'instance exists');
```

Comment:
I'm unsure how this test proves that the search index ignored the .realm.json file. shouldn't you try to load the `.realm.json` file from the search index and assert that it is not there?

Reply:
Updated.